hw/ip/csrng/dv/cov/csrng_v2s_exclusions.vRefine

```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<refinement-file-root>
```

Comment:
Suggest to put the copyright header as comment between lines 1 and 2 (like we do for other refine files)